tests/middleware/test_wsgi.py

```diff
@@ -34,7 +55,7 @@ def echo_body(environ: Environ, start_response: StartResponse) -> List[bytes]:
     return [output]
 
 
-def raise_exception(environ: Environ, start_response: StartResponse) -> RuntimeError:
+def raise_exception(environ: Environ, start_response: StartResponse) -> List[bytes]:
```

Comment:
I understand why you did this, but I don't think it's the right type. It should be `NoReturn` :thinking:

Reply:
From the typechecker perspective WSGIMiddleware expects a function that has `List[bytes]` as return type, if it aways fails into exception is not that important.

This type error started to be more clear because the a2wsgi WSGIMiddleware is less permissive in terms of functions it accepts.